ckanext/unaids/theme/templates/package/releases/list.html

```diff
@@ -17,25 +17,21 @@
         </p>
     {% else %}
         <div class="btn-group">
-            <a href="{{ urls.create_from_latest_version }}" class="btn btn-danger">
-                <span>{{ _('Add Release') }}</span>
+            <a href="{{ urls.create_from_latest_version }}" class="btn btn-primary">
+                <i class="fa fa-plus-square"></i> <span>{{ _('Add Release') }}</span>
             </a>
-            <button type="button" class="btn btn-danger dropdown-toggle" data-toggle="dropdown">
+            <button type="button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown">
                 <span class="caret"></span>
             </button>
             <ul class="dropdown-menu">
                 <li>
                     <a href="{{ urls.create_from_latest_version }}">
-                        <span>{{ _('From the') }} </span>
-                        <u>{{ _('latest version') }} </u>
-                        <span>{{ _('of the dataset') }}</span>
+                        <span>{{ _('From the latest version of the dataset') }}</span>
                     </a>
                 </li>
                 <li>
                     <a href="{{ urls.create_from_an_older_version }}">
-                        <span>{{ _('From a') }} </span>
-                        <u>{{ _('past version') }} </u>
-                        <span>{{ _('of the dataset') }}</span>
+                        <span>{{ _('From a past version of the dataset') }}</span>
```

Comment:
FYI This fixed another translation issue too:
Older version would make translator to deal with 3 separate strings:
1. From a
2. past version
3. of the dataset
This would be really hard to deal with. If you use _('From a <u>past version</u> of a dataset') you present the translator with a  bit of html (which is totally fine) but you also give a full context to them.